trade-finance-manager-ui/templates/utilisation-reports/record-corrections/check-the-information.njk

```diff
@@ -63,7 +63,7 @@
           text: "Provide more information"
         },
         value: {
-          text: additionalInfo
+          html: (additionalInfo | replace('\n', '<br>'))
```

Comment:
could this come from the UI/helper so we don't do logic in nunjucks files?

Reply:
👍 Have made it a custom nunjucks filter so that it can be reused, following the pattern of pre-existing replace filters.